runtime/runtime.go

```diff
@@ -2369,8 +2362,7 @@ func (r *interpreterRuntime) newAuthAccountContractsChangeFunction(
 				context.SetCode(context.Location, string(code))
 
 				panic(fmt.Errorf(
-					"invalid %s: the name argument must match the name of the declaration"+
-						"name argument: %q, declaration name: %q",
+					"invalid %s: the name argument must match the name of the declaration: got %q, expected %q",
```

Comment:
The error message was missing a dot between the two sentences (see `+` operator). I fixed this and also made it consistent with the other error messages (got/expected)